docs/src/components/TailwindClassnames/index.tsx

```diff
@@ -52,6 +69,49 @@ const getCategory = (name: string) => {
   return "palette";
 };
 
+const transformColors = (colors: Record<string, string>) => {
+  return Object.entries(colors).reduce((acc, [category, value]) => {
+    if (typeof value === "object") {
+      Object.entries(value).forEach(([name, color]) => {
+        acc[`${category}-${name}`] = color;
+      });
+    } else {
+      acc[category] = value;
+    }
+
+    return acc;
+  }, {});
+};
+
+const CategoryDescriptions = ({ category }: { category: Category }) => {
+  return (
+    <>
+      {category === "Colors" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used with Tailwind classes that apply a color property (eg: `text-blue-dark`,
+          `bg-red-light`, etc).
+        </Text>
+      )}
+      {category === "Screens" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used as a Tailwind prefix for defining media queries (eg: `sm:opacity-1`,
+          `tb:hidden`, etc).
+        </Text>
+      )}
+      {category === "Spacing" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used as a Tailwind prefix for defining margin and padding (eg: `m-1`, `p-2`, etc).
```

Comment:
```suggestion
          To be used as a Tailwind prefix for defining margin and padding (eg: `m-sm`, `p-xl`, etc).
```

We should use our tokens 😄

Reply:
I've just copied your suggestion :D

Reply:
Hm, no 🤔 😅 
<img width="752" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/e8141cd2-0e0f-44cc-bde7-4367a1e948e0">

Reply:
Hmmm, maybe copilot messed that up :D